caravel/viz.py

```diff
@@ -172,12 +172,9 @@ def get_df(self, query_obj=None):
             raise Exception("No data, review your incantations!")
         else:
             if 'timestamp' in df.columns:
-                if timestamp_format == "epoch_s":
+                if timestamp_format in ("epoch_s", "epoch_ms"):
```

Comment:
I didn't explain very clearly in my previous comment.

Pandas doesn't need to guess whether it's in seconds or milliseconds here after my change, because the data has already be converted to datetime type.